juju-openstack-controller-example.sh

```diff
@@ -7,6 +7,13 @@ fi
 
 set -ux
 
+type openstack
+if [ $? == 1 ]
```

Comment:
Our convention is to do the if, test, and then all on the same line:
if [sometest]; then